cli/tests/data/help/task.txt

```diff
@@ -30,10 +41,14 @@ Options:
         Print the last N lines. The default is 10 lines.
     --long
         Print full Mesos sandbox file attributes.
+    --pty
```

Comment:
If we are dropping it other places, we should also drop it here. it should also be `--tty` though, not `--pty`.